sql/ddl_internal.sql

```diff
@@ -209,8 +212,14 @@ BEGIN
 
     column_type = _timescaledb_internal.dimension_type(main_table, column_name, num_slices IS NULL);
 
+    IF interval_length IS NOT NULL THEN
+        interval_length_actual = _timescaledb_internal.time_interval_specification_to_internal_with_default_time(
+            column_type, interval_length, 'interval_length',
+            ignore_interval_too_small=>hide_interval_warnings);
```

Comment:
why is one parameter called  ignore_interval_too_small and the other hide_interval_warnings. Aren't these the same?

Reply:
I can change that